### ISSUE TITLE
How to handle when a term doesn't exists?

### ISSUE THREAD

**Issue by ppazos** [Philipp15b/php-i18n#67]
When a term doesn't exists I'm getting an exception "Fatal error: Uncaught Error: Undefined class constant".

It happens when two lang files exist but one has missing translations.

**Comment by Philipp15b** [comment]
Yes, if the translation string is not available in a language file then the class does not contain the corresponding string. This is intended behaviour.

You could check if a term is defined in the current language using something like [PHP's defined](https://www.php.net/manual/en/function.defined.php).
Hope that helps.
